config/environments/development.rb

```diff
@@ -43,5 +43,5 @@
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
-  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
```

Comment:
I prefer we add the listen gem back in, because this is meant for do auto-reload and such.